docs/deploy-rbd.md

```diff
@@ -196,6 +198,24 @@ The Helm chart is located in `charts/ceph-csi-rbd`.
 
 [See the Helm chart readme for installation instructions.](../charts/ceph-csi-rbd/README.md)
 
+## Read Affinity using crush locations for RBD volumes
+
+Ceph CSI supports mapping RBD volumes with krbd options
+`"read_from_replica=localize,crush_location=type1:value1|type2:value2"` to achieve
+reads from the most local serving OSD.
+Refer [krbd-options](https://docs.ceph.com/en/latest/man/8/rbd/#kernel-rbd-krbd-options)
+for more details.
+
+This can be enabled by adding labels to Kubernetes nodes in the format `"topology.io/zone=east"`
+and passing cmdline arguments `"--enable-read-affinity=true"` and
+`"--crush-location-labels=topology.io/zone,topology.io/region"`
+to Ceph CSI RBD daemonset pod "csi-rbdplugin" container, resulting in Ceph CSI adding
+`"--options read_from_replica=localize,crush_location=zone:east"` krbd options during
+rbd map operation.
+If enabled, this option will be added to all RBD volumes mapped by Ceph CSI.
```

Comment:
It would be nice to also note that, for this setting to be effective, the OSDs should be assigned to the same  zones, regions, etc respectively on the cluster side (in the CRUSH map).  Just declaring that a particular node belongs to a particular zone on the client side is insufficient.

Reply:
I've modified the first sentence to elaborate on it, 
But users should find more info on ceph krbd map options site which is linked in second sentence.